src/components/GlobalPreferences/Notifications/NotificationsLogin.js

```diff
@@ -160,13 +160,15 @@ export default function NotificationsLogin({
         different browser session or device to access your subsciptions. This
         process doens’t require a password, just for you to confirm your email
         address.`
-          : `Receive email notifications for new events in this organization. For
-        example, whenever a new vote is created or when funds are transferred,
-        you’ll get an email informing you of the latest activity in your organization.
-        You will be asked to enter with your email address whenever using a
-        different browser session or device to access your subsciptions. This
-        process doens’t require a password, just for you to confirm your email
-        address whenever you want to sing in.`}
+          : `Receive email notifications for new app events. For example,
+        whenever a new vote is created or when tokens are issued, you’ll get
```

Comment:
```suggestion
        whenever a new vote is created or when funds are transferred, you’ll get
```

We can not detect token transfers yet.